draft-ietf-quic-transport.md

```diff
@@ -2679,7 +2679,9 @@ endpoint receives.
 This design relies on the peer always sending a connection ID in its packets so
 that the endpoint can use the connection ID from a packet to reset the
 connection.  An endpoint that uses this design cannot allow its peers to send
-packets with a zero-length destination connection ID.
+packets with a zero-length destination connection ID and need to either use
+the same connection ID length for all connections or encode the length of
+the connection ID such that it can be recovered without state.
```

Comment:
Late to the party, but this is pretty hard to read. I believe that it is intended to say:

(1) must not use a zero-length CID
(2) must  have CID length be either (a) constant or (b) encoded in the CID itself

But the use of "cannot allow" might instead say you can't do both of these together instead. I would instead suggest.

"an endpoint that uses this design MUST either use the same connection ID length for all connections or encode the length of the connection ID such that it can be recovered without state. In addition, it MUST NOT provide zero-length destination connection IDs to its peers".